loadsave.c

```diff
@@ -132,7 +151,20 @@ LOAD()
 		RAM[STATUS] = 0;
 		a = 0;
 	} else {
-		FILE *f = fopen(filename, "rb");
+        
+        //use correct file location for ios
+        //use correct file location for ios
+        #if __APPLE__ && (TARGET_IPHONE_SIMULATOR || TARGET_OS_IPHONE)
+        char fullFileName[256];
+        //concatenating the path string returned from HOME
+        strcpy(fullFileName,getenv("HOME"));
+        strcat(fullFileName,"/Documents/");
+        strcat(fullFileName,filename);
+        FILE *f = fopen(fullFileName, "rb");
+        #else
+        FILE *f = fopen(filename, "rb");
```

Comment:
I'd suggest to do as I suggested above: `char full_filename[256]` and strcpy/strcat in the iOS case, `char *full_filename = filename` in the default case; then `FILE *f = fopen(full_filename, "rb");` to minimize code duplication between the paths.

Reply:
👍